packages/courDeCassation/src/scripts/importAllDocumentsFromSderSinceOrBetween.ts

```diff
@@ -10,20 +10,27 @@ import { environmentType } from '@label/core';
   const backend = buildBackend(environment, settings);
 
   backend.runScript(
-    () => importAllDocumentsFromSderSince(days, byDateCreation, environment),
+    () =>
+      importAllDocumentsFromSderSinceOrBetween(
+        days,
+        byDateCreation,
+        environment,
```

Comment:
Il faut passer la variable `to` en argument sinon elle n'ira jamais dans la fonction ;)